### DISCUSSION TITLE
thoughts on adding daisyUI as a dependency vs dev-dependency in nuxt3 (or others)?

### DISCUSSION THREAD

**Discussion by AliQ80** [saadeghi/daisyui#968] [General]
Hi,

I know the difference as per npm documentation 

"dependencies": Packages required by your application in production.
"devDependencies": Packages that are only needed for local development and testing.

I'm not an expert, so to my understanding if tailwind itself is a dev-dependency (as per their setup instructions), shouldn't daisyUI be the same? or is there a specific reason that daisyUI should be a dependency?

thank you

**Comment by saadeghi** [comment]
For almost every project, daisyUI (and Tailwind CSS) can be `dev-dependency` because they only generate the CSS file on build step.  

However some people need to import [files from daisyUI package](https://unpkg.com/browse/daisyui@2.19.1/) directly to their project files (to use color functions, generate themes live, etc...) in this cases it should be `dependency `

**Reply by AliQ80** [reply]
Thanks for the reply, that's good info

Honestly I love daisyUI.
